crates/starlight/src/bytecompiler.rs

```diff
@@ -1606,6 +1606,11 @@ impl ByteCompiler {
                     self.handle_builtin_call(call)?;
                 }
             }
+            x if is_codegen_plugin_call(self.rt, x) => {
```

Comment:
Codegen plugins should be enabled via runtime options or when `self.builtins` is true since it is not the safest feature.